_data/banking.yml

```diff
@@ -630,7 +630,6 @@ websites:
       tfa: Yes
       sms: Yes
       email: Yes
-      doc: https://www.macu.com/home/services/online-banking/new-online-branch
```

Comment:
The documentation you provided says "Receive MFA PINs through text, email or automated phone call."

Reply:
```suggestion
      phone: yes
```